Description: Tests checking "std::basic_string<char>" return value by default.
 But llvm in Debian is returning C++11 alias of std::basic string. This patch
 is making subcommand_test.py to check alias instead of default value.

--- a/ycmd/tests/subcommands_test.py
+++ b/ycmd/tests/subcommands_test.py
@@ -271,12 +271,12 @@
     [{'line_num' : 13, 'column_num': 3} , 'Unknown type'], # sic
     # on string (string)
     [{'line_num' : 13, 'column_num': 8} ,
-                                  'string => std::basic_string<char>'], # sic
+                                  'string => std::__cxx11::basic_string<char>'], # sic
     # on a (std::string)
     [{'line_num' : 13, 'column_num': 15} ,
-                                  'std::string => std::basic_string<char>'],
+                                  'std::string => std::__cxx11::basic_string<char>'],
     [{'line_num' : 14, 'column_num': 16} ,
-                                  'std::string => std::basic_string<char>'],
+                                  'std::string => std::__cxx11::basic_string<char>'],
 
   # cursor on decl for refs & pointers
     [{'line_num' : 27, 'column_num': 3} ,  'Foo'],
